phpcs.xml.dist

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="WordPress Coding Standards" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/squizlabs/PHP_CodeSniffer/master/phpcs.xsd">
+
+  <!-- See https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml -->
+  <!-- See https://github.com/WordPress/WordPress-Coding-Standards/blob/develop/.phpcs.xml.dist -->
+
+  <!-- Set a description for this ruleset. -->
+  <description>Coding standards used for checking the Twenty Twenty theme.</description>
+
+  <!-- Exclude patterns -->
+  <exclude-pattern>/vendor/*</exclude-pattern>
+  <exclude-pattern>/node_modules/*</exclude-pattern>
+
```

Comment:
* You're missing the `file` directive. The Composer script as-is in composer will error with a "No files to scan provided" error unless the `.` is explicitly passed (which Travis does, but shouldn't need to).
    ```xml
    <file>.</file>
    ``` 
* What about adding:
    - `basepath`
    - `parallel`
    - `cache`
    - and possibly `extensions`

Reply:
I like this idea. Since #35 aims to add linting and build process for JS and CSS, phpcs should just focus on PHP 🙂 :+1:

Reply:
If I remember correctly, there are actually some WPThemeReview sniffs which examine CSS and possibly JS as well, so be careful with that. All the same, making explicit that either only PHP files are scanned or that all three extensions are scanned, is not a bad thing in itself.

Reply:
A follow up PR should make switch this to PHP files please @dingo-d